prow/plugins/trigger/ic.go

```diff
@@ -26,7 +26,10 @@ import (
 	"k8s.io/test-infra/prow/plugins"
 )
 
-var okToTest = regexp.MustCompile(`(?m)^(@k8s-bot )?ok to test\r?$`)
+// The commands can have additional commands behind them, but not before
+// Any additional comment and white spaces before the commands will NOT trigger the commands
+// The commands can have comma appended, e.g., "ok to test," "/ok-to-test," "/test," "/test unit,"
+var okToTest = regexp.MustCompile(`((?m)^(@k8s-bot )?ok to test\r?$|(?m)^/ok-to-test\r?$)`)
```

Comment:
The existing codes have the regex as `(?m)^(@k8s-bot )?ok to test\r?$`. i left it intact.